ui/src/interfaces/stats.interface.ts

```diff
@@ -0,0 +1,41 @@
+export interface leaderBoardEntry {
+  id: string;
+  name: string;
+  rank: number;
+  rating: number;
+//   wins: number;
```

Comment:
it's upcoming right ?

Reply:
Yes I asked Mouad to add wins and losses to the getLeaderboard request. I would have to make individual requests for every player in the leaderboard which seems like a little overkill.